linode/manager
# M3-5409: API methods for Account Agreements

## Description
- Add API methods and query hook for account agreements
- Add wrapper for class-based components

## How to test
- Go to `linodes/create` and check the browser dev tool Console. You should see the state of all agreements
    - Go to the browser's dev tool Network tab, then click `Create Linode`. You should see an agreements request sent
